openmetadata-service/src/main/resources/json/data/ResourceDescriptors.json

```diff
@@ -57,6 +57,20 @@
       "EditLineage"
     ]
   },
+  {
+    "name" : "query",
+    "operations" : [
+      "Create",
+      "Delete",
+      "ViewAll",
+      "ViewQueries",
+      "EditAll",
```

Comment:
Do we need ViewQueries given the entity is a Query entity?

Reply:
As mentioned, if we ever want to be able to create Queries from UI then this will be needed, otherwise, if we are only aiming for ingestion to put Queries then I can change as per that, let me know!